scripts/kfctl.sh

```diff
@@ -486,6 +486,9 @@ main() {
       kubectl delete ns/${K8S_NAMESPACE}
       while kubectl get ns/${K8S_NAMESPACE}; do
         echo "namespace ${K8S_NAMESPACE} not yet deleted. sleeping 10 seconds..."
+        # wait for 100s at most
+        # https://github.com/kubeflow/kubeflow/issues/2408
+        ((c++)) && ((c==10)) && break
```

Comment:
We should give message ```delete failed``` instead of ```successfully deleted``` if we break here due to timeout

Reply:
done